modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro

```diff
@@ -1,5 +1,54 @@
 definition {
 
+	macro addNotificationEmailViaAPI {
```

Comment:
Please follow my original instructions and name it `addEmailNotificationTemplateViaAPI`